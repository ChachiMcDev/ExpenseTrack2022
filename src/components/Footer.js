import React from 'react';


const Footer = () => (
    <header>
        <h1>Im the footer</h1>
        <div className="menu">social media and address and what not</div>
    </header>
)


export default Footer