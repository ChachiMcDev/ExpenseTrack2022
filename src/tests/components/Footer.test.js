import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';


test('should render NotFoundPage correctly', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();

})