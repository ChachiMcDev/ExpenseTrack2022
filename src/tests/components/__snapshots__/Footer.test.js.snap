// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<header>
  <h1>
    Im the footer
  </h1>
  <div
    className="menu"
  >
    social media and address and what not
  </div>
</header>
`;
